### ISSUE TITLE
Alt text missing on multiple images

### ISSUE THREAD

**Issue by ibrahimghani** [ibrahimghani/DynamicTechUAT#8]
### Defect description
```

### Steps to reproduce

### Actual results

alt text missing on images
```
Executed by: Ibrahim

### Test case information
Test: **Accessibility Check**
Priority: **high**
Tags: 
Link to:

### Steps
1. Use a screen reader to navigate through the website
2. Check for alt text on images, ARIA labels on interactive elements, and logical tab navigation.

### Failure attachments

---

🔗 Linked to **Testrun ['Accessibility Check '](https://app.testomat.io/projects/dynamictech-website-phase-1/runs/89a0b7b5/report/test/53177975)** in Testomat.io

**🔴 FAILED** in [UAT - Phase 1 Sanity Check](https://app.testomat.io/projects/dynamictech-website-phase-1/runs/89a0b7b5)

> alt text missing on images
Environment: Windows,iOS
Finished At: 29 Aug 2024 13:41
